server/modules/survey/service/export/jobs/CSVDataExtraction.js

```diff
@@ -0,0 +1,17 @@
+import Job from '@server/job/job'
+
+import * as SurveyRdbManager from '@server/modules/surveyRdb/manager/surveyRdbManager'
+
+export default class CSVDataExtraction extends Job {
+  constructor(params) {
+    super('CSVDataExtraction', params)
+  }
+
+  async execute() {
+    const { surveyId } = this.context
+
+    const { exportDataFolderName, dir } = await SurveyRdbManager.fetchEntitiesDataToCsvFiles({ surveyId }, this.tx)
```

Comment:
we should somehow get some feedback on the status of the export, by passing a callback or something to that method and update processed items of the job accordingly

Reply:
not now, maybe in the future when the job could be more complex